src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -15,9 +17,8 @@ public class Line extends BaseEntity {
     @Column(unique = true)
     private String name;
     private String color;
-
-    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
-    private List<Section> sections = new ArrayList<>();
+    @Embedded
+    private Sections sections = new Sections(new ArrayList<>());
```

Comment:
```suggestion
    private Sections sections = new Sections();
```
적절한 생성자를 만들어서 이렇게 사용할 수 있으면 어떨까요 :)

Reply:
다음미션 때 반영하겠습니다!